packages/menu/src/Menu.ts

```diff
@@ -587,22 +587,21 @@ export class Menu extends SizedMixin(SpectrumElement, { noDefaultSize: true }) {
 
     protected navigateBetweenRelatedMenus(event: KeyboardEvent): void {
         const { key } = event;
+        event.stopPropagation();
```

Comment:
Why did we removed `event.stopPropagation();` from line 597 and 605 and added it here?

Reply:
because now we want to make sure that the keyboardEvents don't surface upto the rovingtabIndexController once the menu is open otherwise it leads to an in-accessible state of action-group.